apps/astro-blog/src/app.css

```
@@ -1,5 +1,12 @@
 @import 'tailwindcss';
 
+:root {
+  /* Shared typography roles for editorial pages and components. */
+  --font-display: 'Times New Roman', serif;
+  --font-body: 'Inter', 'Noto Sans JP', sans-serif;
+  --font-meta: 'SF Mono', Monaco, Consolas, 'Liberation Mono', 'Courier New', monospace;
```

Stylelint (17.14.0)

[error] Expected "Monaco" to be "monaco" (value-keyword-case)

(value-keyword-case)